<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qtel: Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="TabWidget2">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="user_info_tab">
      <attribute name="title">
       <string>User Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>User Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel1">
            <property name="text">
             <string>Callsign</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="my_callsign"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="TextLabel2">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLineEdit" name="my_name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel5">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="my_password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="TextLabel6">
            <property name="text">
             <string>Retype password</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="my_vpassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLabel3">
            <property name="text">
             <string>Location</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="my_location"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="TextLabel4">
            <property name="text">
             <string>Info
Message</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="4">
           <widget class="QTextEdit" name="my_info"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="directory_server_tab">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Directory Server Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Bind address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel7">
            <property name="text">
             <string>Servers</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="directory_servers">
            <property name="toolTip">
             <string>A space separated list of the hostname or IP address of the EchoLink directory servers to use</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel2_2">
            <property name="text">
             <string>Refresh time</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSpinBox" name="list_refresh_time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Time, in minutes, between station list refreshes</string>
            </property>
            <property name="suffix">
             <string>min</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>324</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLabel3_2">
            <property name="text">
             <string>Start as busy</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="start_as_busy_checkbox">
            <property name="toolTip">
             <string>Register as busy on program startup</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>365</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="bind_address">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A bind address can be specified to force Qtel to use a specific network interface if your computer has more than one. Specify the IP address of the interface to bind to.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>EchoLink Proxy Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="proxy_enabled_label">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="proxy_enabled">
            <property name="toolTip">
             <string>If enabling EchoLink proxy you must supply a proxy server hostname or IP address.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="proxy_server_label">
            <property name="text">
             <string>Server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="proxy_server">
            <property name="toolTip">
             <string>The hostname or IP address of the EchoLink proxy server to use</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="proxy_port_label">
            <property name="text">
             <string>TCP Port</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="proxy_port">
            <property name="toolTip">
             <string>The TCP port to connect to on the EchoLink proxy server, default 8100</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>8100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>425</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="proxy_password_label">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="proxy_password">
            <property name="toolTip">
             <string>The EchoLink PROXY password, not your normal EchoLink password. Leave empty for public proxy.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>42</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sound_tab">
      <attribute name="title">
       <string>Sound</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Sound Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Mic audio device</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mic_audio_device">
              <property name="toolTip">
               <string>The audio device to use (e.g. alsa:default)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="textLabel1_4">
              <property name="text">
               <string>Speaker audio device</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="spkr_audio_device">
              <property name="toolTip">
               <string>The audio device to use (e.g. alsa:default)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="card_sample_rat_label">
              <property name="text">
               <string>Sampling Rate</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QComboBox" name="card_sample_rate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>308</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="textLabel2">
              <property name="text">
               <string>Full duplex</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="use_full_duplex">
              <property name="toolTip">
               <string>Check this check box to enable full duplex operation</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="textLabel1_2">
              <property name="text">
               <string>Connect Sound</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="connect_sound">
                <property name="toolTip">
                 <string>The connect sound to use on incoming EchoLink connections</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="connect_sound_browse_button">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>135</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qso_tab">
      <attribute name="title">
       <string>QSO</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>QSO Dialog Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_3">
            <property name="text">
             <string>Chat encoding:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="chat_encoding"/>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>296</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>232</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="Spacer3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ok_button">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancel_button">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>my_callsign</tabstop>
  <tabstop>my_name</tabstop>
  <tabstop>my_password</tabstop>
  <tabstop>my_vpassword</tabstop>
  <tabstop>my_location</tabstop>
  <tabstop>my_info</tabstop>
  <tabstop>TabWidget2</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>directory_servers</tabstop>
  <tabstop>list_refresh_time</tabstop>
  <tabstop>start_as_busy_checkbox</tabstop>
  <tabstop>mic_audio_device</tabstop>
  <tabstop>spkr_audio_device</tabstop>
  <tabstop>use_full_duplex</tabstop>
  <tabstop>connect_sound</tabstop>
  <tabstop>connect_sound_browse_button</tabstop>
  <tabstop>chat_encoding</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connect_sound_browse_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>browseConnectSound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseConnectSound()</slot>
 </slots>
</ui>
